confmap/confmap.go

```diff
@@ -169,6 +169,13 @@ func (l *Conf) Merge(in *Conf) error {
 	return l.k.Merge(in.k)
 }
 
+// MergeAppend merges the input given configuration into the existing config.
+// Note that the given map may be modified.
+// Note: MergeAppend merges the lists in a name-aware fashion.
+func (l *Conf) MergeAppend(in *Conf, paths []string) error {
```

Comment:
```suggestion
func (l *Conf) mergeAppend(in *Conf, paths []string) error {
```
Could we unexport this for now? Since we only call this method within this package, it would be nice not to offer it externally until we're positive we want this to be part of the public API. confmap is a stable module, so once something is made public, we can't break it if we find a better solution later.

Reply:
Sure. That makes sense.